client/dashboard/index.js

```diff
@@ -26,23 +27,25 @@ const CustomizableDashboard = lazy( () =>
 	import( /* webpackChunkName: "customizable-dashboard" */ './customizable' )
 );
 
-const ProfileWizard = lazy( () =>
-	import( /* webpackChunkName: "profile-wizard" */ '../profile-wizard' )
-);
-
 class Dashboard extends Component {
 	render() {
 		const { path, profileItems, query } = this.props;
+		const {
+			completed: profileCompleted,
+			skipped: profileSkipped,
+			step: profilerStep,
+		} = profileItems;
 		if (
 			isOnboardingEnabled() &&
-			! profileItems.completed &&
+			! profileCompleted &&
+			! profileSkipped &&
 			! window.wcAdminFeatures.homescreen
 		) {
-			return (
-				<Suspense fallback={ <Spinner /> }>
-					<ProfileWizard query={ query } />
-				</Suspense>
+			const lastStep = profilerStep ? `&step=${ profilerStep }` : '';
+			const href = getAdminLink(
+				`admin.php?page=wc-admin&path=/profiler${ lastStep }`
 			);
+			getHistory().push( href );
```

Comment:
I think this should actually be:

```suggestion
			getHistory().push( getNewPath( {}, '/profiler', {} ) );
```

Otherwise the URL becomes `https://wp.local/wp-admin/https://wp.local/wp-admin/admin.php?page=wc-admin&path=/profiler&step=store-details` in my local install.